controllers/articles.js

```diff
@@ -4,8 +4,8 @@ const {
   selectArticles,
 } = require("../models/articles");
 
-exports.getArticles = (req, res, next) => {
-  selectArticles()
+exports.getArticles = ({ query }, res, next) => {
```

Comment:
Nice use of parameter destructuring here as long as you're comfortable you will never need anything else on the request - which you don't really here!